onnx/defs/generator/defs.cc

```diff
@@ -51,6 +99,60 @@ ONNX_OPERATOR_SET_SCHEMA(
             return;
           }
 
+          if (nullptr != value_int) {
+            // OpSchema::Verify check ensures that the attribute value has_i():
+            if (!value_int->has_i())
+              fail_shape_inference("Attribute 'value_int' expect an integer.")
```

Comment:
This is indeed a bit redundant with the type definition of value_int, but maybe redundancy is a good thing? 😅